package.json

```diff
@@ -220,6 +222,16 @@
                 "title": "Disable Remote Debugging",
                 "category": "Azure App Service"
             },
+            {
+                "command": "appService.StartSsh",
+                "title": "Start SSH",
+                "category": "Azure App Service"
+            },
+            {
+                "command": "appService.StopSsh",
+                "title": "Stop SSH",
```

Comment:
I'm not sure how I feel about 'Stop SSH'. It almost feels more intuitive to delete the terminal in VS Code or to just send `ctrl+c` to the terminal. Thoughts? It would help reduce the number of context menu items

Reply:
Yeah, the problem is the only way to end SSH is to actually trash the terminal (unless I use Tasks).  I don't feel like that garbage icon is super intuitive to stop SSH.  If you Ctrl+C or type `exit`, only the SSH connection is killed, but as the terminal remains, there is no way for us to detect it.